Core/include/Acts/Definitions/Alignment.hpp

```diff
@@ -32,17 +32,11 @@ enum AlignmentIndices : unsigned int {
   eAlignmentSize,
 };
 
-/// Underlying fundamental Scalar type for alignment parameters.
-using AlignmentScalar = ActsScalar;
```

Comment:
We might have to check if we need a dedicated scalar setting if we want to use auto-diff for the alignment?

Reply:
I expect that using auto-diff for alignment will be handled similar to its use in the stepper code, i.e. the implementation is templated on the scalar type which defaults to `ActsScalar`. This is orthogonal to having a separate `AlignmentScalar` typedef.